watertap/examples/flowsheets/case_studies/BSM2/BSM2_flowsheet2.py

```diff
@@ -0,0 +1,422 @@
+#################################################################################
```

Comment:
@luohezhiming I am not sure where this is coming from. I don't think it is from main because I reorganized this and renamed this BSM2 flowsheet, so you should start by making sure you're using the most up-to-date work